src/main/preprocessing/preprocessing.py

```diff
@@ -84,7 +99,11 @@ def preprocess_data(path):
     for folder in folders:
```

Comment:
And also let's return the new data path to the folder, all preprocessed files are written to